databases/schemas/friends.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE IF NOT EXISTS friends (
+	userid TEXT NOT NULL,
+	friend TEXT NOT NULL,
+	previous_name TEXT,
+	change_date INTEGER,
+	last_login INTEGER,
+	PRIMARY KEY (userid, friend) ON CONFLICT IGNORE
```

Comment:
Keep the schema as flexible as possible. Move `ON CONFLICT IGNORE` to inserts/updates where you specifically want that functionality